Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -779,7 +787,9 @@ def SwapKeepAndDrop(l):
 REPACKRAWEventContent.outputCommands.extend(HLTriggerRAW.outputCommands)
 approxSiStripClusters.toModify(REPACKRAWEventContent,
                                outputCommands = REPACKRAWEventContent.outputCommands+[
-                                   'keep *_SiStripClusters2ApproxClustersHLT_*_*'
+                                   'keep *_hltSiStripClusters2ApproxClusters_*_*',
+                                   'drop FEDRawDataCollection_rawDataRepacker_*_*',
+                                   'keep *_rawPrimeDataRepacker_*_*'
```

Comment:
Would this also work?
```suggestion
                                   'keep FEDRawDataCollection_rawPrimeDataRepacker_*_*'
```

Reply:
good point, we should specify the type.  Will wait a bit for more comments and commit that, also for  `'keep *_hltSiStripClusters2ApproxClusters_*_*',`